proposals/collection-expression-arguments.md

```diff
@@ -300,10 +298,27 @@ If the target type is a type with a *create method*, then:
   * If the constructor has a `params` parameter, the invocation may be in expanded form.
 * Otherwise, a binding error is reported.
 
-If the target type is a *dictionary interface* with type arguments `K` and `V`, and the *argument list* is not empty, then:
-* If the *argument list* is a single value, implicitly convertible to `IEqualityComparer<K>`, and optionally with name `comparer`, a dictionary instance is constructed with that key comparer value.
+If the target type is an *interface type*, then:
+* [*Overload resolution*](https://github.com/dotnet/csharpstandard/blob/standard-v7/standard/expressions.md#1264-overload-resolution) is used to determine the best instance constructor from the *argument list* from the following candidate signatures:
+  * If the target type is `IEnumerable<E>`, `ICollection<E>`, or `IList<E>`, the candidates are:
```

Comment:
It's only the mutation interfaces that we guarantee building concrete `List<T>` for. `IEnumerable<T>` is in with `IReadOnlyCollection<T>` etc where we use a compiler-generated wrapper to prevent mutations. If the reason for exposing `new(int capacity)` is that users know they are getting `List<T>`, that would not apply to the `IEnumerable<T>` target. If there is a separate reason to expose capacity that applies to `IEnumerable<T>`, I would expect that same reason to apply to `IReadOnlyCollection<T>` etc.